src/main/java/com/LP2/EventScheduler/service/connection/ConnectionServiceImpl.java

```diff
@@ -110,4 +110,18 @@ public MessageResponse acceptInvitation(UUID invitationId, User authUser) {
 
         return new MessageResponse("Invitation accepted");
     }
+
+
+    @Transactional
+    @Override
+    public MessageResponse removeConnection(UUID connectionId, User authUser) {
+        Connection connection = connectionRepository.findById(connectionId)
+                .orElseThrow(() -> new ConnectionNotFoundException("Connection not found"));
+        if (!connection.getConnector().equals(authUser) && !connection.getConnecting().equals(authUser)) {
```

Comment:
Al utilizar el `.equals()` no compares objetos enteros, compara sus `ID` o propiedad que sea unica
```suggestion
        if (!connection.getConnector().getId().equals(authUser.getId()) && !connection.getConnecting().getId().equals(authUser.getId())) {
```